app/helpers/application_helper.rb

```diff
@@ -291,10 +292,10 @@ def realization_path(realization)
   def first_course_independent?
     current_user.administrated_courses
                 .natural_sort_by(&:title)
-               &.first&.term_independent
+                &.first&.term_independent
   end
 
-  def get_announcements
```

Comment:
Since we include the `ApplicationHelper` in `lecture.rb`, renaming this class to `announcements` has the consequence that this method overrides the `announcements` method of the `lecture` model which is given by the `:has_many announcements` relation.

Reply:
Oh, glad you caught this! I went through every method of the `application_helper.rb` and found no method that is overwritten in `lecture.rb`. Just a few things that shouldn't be a problem though:

- helper has `active(value)` method, in lecture we use `Term.active`
- helper has `lecture_media` method, we use this as variable name in lecture (but locally scoped, so should be fine), same for `course_media`

The `ApplicationHelper` is also included in `course.rb`, `item.rb`, `medium.rb` and `user.rb`. Shouldn't we also find duplicate naming not just in `lecture.rb` but in these other files as well? If yes, I could do this.

Reply:
> Did you mean "method" instead of "class"?

Yes.

Reply:
I do not really think that is really necessary (in contrast to the lectures_helper there has not been a related change here), but feel free to do so if you think it is better.

Reply:
You're right, I've briefly checked the mentioned files and there seems to be no conflict (I searched for "announce" and these files do not even include that word).